api/app/resources/bookings/exam/exam_list.py

```diff
@@ -31,8 +31,13 @@ class ExamList(Resource):
     def get(self):
         try:
             csr = CSR.find_by_username(g.oidc_token_info['username'])
+            search_arguments = request.args
+
+            if search_arguments:
```

Comment:
Is it possible to refactor this like:

```
exams = Exam.query.filter(Exam.deleted_date.is_(None)).filter_by(office_id=csr.office_id

if request.args:
    exams = exams.filter_by(**request.args)
```

Reply:
Is this a different way to skin the cat, or just accomplishing it in fewer lines?

Reply:
Yea basically. Less code best code

Reply:
I'll give it a shot. Give me some time.

Reply:
cool, didn't know you could filter a second time. this might be helpful later.

I'll get this updated in a few minutes.